src/main/kotlin/by/architecture_map/belarus/service/ConstructionImageService.kt

```diff
@@ -0,0 +1,14 @@
+package by.architecture_map.belarus.service
+
+import by.architecture_map.belarus.entity.ConstructionImage
+
+interface ConstructionImageService {
+
+    fun create(image: ConstructionImage): ConstructionImage
+    fun getRandomAndUniqueImages(gotImagesId: String?): MutableList<ConstructionImage>
+    fun getByConstructionArchitecturalStyleId(architecturalStyleId: Int): MutableList<ConstructionImage>
+
+    // If you can propose more descriptive name for this method please share <3
+    fun getImagesWithSameArchitecturalStyleByConstructionIdAcrossImagesOfCurrentConstruction(constructionId: Int): MutableList<ConstructionImage>
+    fun deleteById(id: Int): Boolean
```

Comment:
also, regarding this name the same situation. `deleteById` - is good for repository method, but for services we usually use just `delete` or `deleteImage`. you can descripe in Doc comment by what parameter you perform the operation